pkg/antctl/raw/check/installation/command.go

```diff
@@ -0,0 +1,394 @@
+	}
+	t.Log("Deploying echo-same-node service...")
+	svc := newService(echoSameNodeDeploymentName, map[string]string{"name": echoSameNodeDeploymentName}, 80)
+	_, err = t.client.CoreV1().Services(t.namespace).Create(ctx, svc, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	echoDeployment := newDeployment(deploymentParameters{
+		Name:    echoSameNodeDeploymentName,
+		Role:    kindEchoName,
+		Port:    80,
+		Image:   deploymentImage,
+		Command: []string{"/agnhost", "netexec", "--http-port=80"},
+		Affinity: &corev1.Affinity{
+			PodAffinity: &corev1.PodAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								{
+									Key:      "name",
+									Operator: metav1.LabelSelectorOpIn,
+									Values:   []string{clientDeploymentName},
+								},
+							},
+						},
+						TopologyKey: "kubernetes.io/hostname",
+					},
+				},
+			},
+		},
+		Labels: map[string]string{"app": echoSameNodeDeploymentName},
+	})
+	_, err = t.client.AppsV1().Deployments(t.namespace).Create(ctx, echoDeployment, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("unable to create Deployment %s: %s", echoSameNodeDeploymentName, err)
+	}
+	t.Log("Deploying client Deployment...")
+	clientDeployment := newDeployment(deploymentParameters{
+		Name:    clientDeploymentName,
+		Role:    kindClientName,
+		Image:   deploymentImage,
+		Command: []string{"/agnhost", "pause"},
+		Port:    80,
+		Labels:  map[string]string{"app": clientDeploymentName},
+	})
+	_, err = t.client.AppsV1().Deployments(t.namespace).Create(ctx, clientDeployment, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("unable to create Deployment %s: %s", clientDeploymentName, err)
+	}
+
+	t.Log("Deploying echo-other-node Service...")
+	svc = newService(echoOtherNodeDeploymentName, map[string]string{"name": echoOtherNodeDeploymentName}, 80)
+	_, err = t.client.CoreV1().Services(t.namespace).Create(ctx, svc, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	echoOtherNodeDeployment := newDeployment(deploymentParameters{
+		Name:    echoOtherNodeDeploymentName,
+		Role:    kindEchoName,
+		Port:    80,
+		Image:   deploymentImage,
+		Command: []string{"/agnhost", "netexec", "--http-port=80"},
+		Affinity: &corev1.Affinity{
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								{Key: "name", Operator: metav1.LabelSelectorOpIn, Values: []string{clientDeploymentName}},
+							},
+						},
+						TopologyKey: "kubernetes.io/hostname",
+					},
+				},
+			},
+		},
+		Labels: map[string]string{"app": echoOtherNodeDeploymentName},
+	})
+	nodes, err := t.client.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
there is no reason to use `context.TODO()` given that the function takes an actual context as a parameter (`ctx`)